### ISSUE TITLE
Custom cursor has unexpected behavior using recommended dimensions

### ISSUE THREAD

**Issue by ddxt** [godotengine/godot-docs#4554]
**Godot Version**
3.2.stable.custom_build

**Issue Description**
The documentation for custom cursors says that the image has to be less than 256x256 pixels, but it actually has to be much smaller than that, around 32x32. Using a custom cursor with 200x200 pixels causes the cursor to flicker and dissappear, especially near the top of the screen.

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/tutorials/inputs/custom_mouse_cursor.html

**Comment by Calinou** [comment]
@ddxt Which operating system are you using?

**Comment by ddxt** [comment]
@Calinou I'm using Ubuntu 20.10.

I did some more testing to see if I could figure out what was causing the issue, and I found that 128x128 is the larget the cursor can be before it starts flickering. The flickering is also much less visible when the game is run as a windowed application instead of fullscreen. Finally, when taking screenshots or OBS to try and record it, the issue isn't visible. [Here's a video](https://i.imgur.com/Um3JKwA.mp4) I recorded with my phone so you can see what it looks like. The cursor is supposed to be a black box btw, I was just using it to try and troubleshoot.

Hopefully that helps, if you need any more info, let me know.

**Comment by Alex-Sand-0** [comment]
Hi, 
I also observed some flickering behavior on windows with cursors slightly larger than 128x128.
More precisely, I used  Input.SetCustomMouseCursor to change the cursor images, and the flickering appeared when moving the mouse well inside a control with a  MouseDefaultCursorShape not set to 'arrow': Most frames where displaying the correct cursor, but there were a few frames with the image I defined for "arrow". 
Reducing the size of the images (around 64x64) seems to fix the issue.

System: window 10, godot 3.3 (stable, mono, official), running scripts in c#
